.github/workflows/release.yaml

```diff
@@ -0,0 +1,79 @@
+name: create release on new tags
+
+# This workflow is triggered by new version tags (e.g. "1.12.2") on the main branch.
+#
+# Tagging should be automatic after each merged PR release (see the build.yaml workflow). It can
+# also be manually triggered if needed. Note that the tests are not run by this workflow, so
+# release manually at your own risk. If manually tagging the main branch, the tag must match the
+# version set in ./pkg/chartverifier/version/version_info.json.
```

Comment:
Just so I'm clear, when you say this can be manually triggered, you mean by creating a tag?
In other words, there's no `workflow_dispatch` event that this triggers on, and that's normally what I see when a workflow can be manually triggered.

Reply:
That's correct, it can be triggered ~manually~ by manually creating a tag on the main branch with the right format (x.x.x without leading v).

It's not intended to be triggered manually via the GitHub GUI / CLI.

I'll reword the comment.